docs/resources/agent.md

```diff
@@ -48,6 +48,7 @@ resource "kubernetes_pod" "dev" {
 
 - `auth` (String) The authentication type the agent will use. Must be one of: "token", "google-instance-identity", "aws-instance-identity", "azure-instance-identity".
 - `connection_timeout` (Number) Time in seconds until the agent is marked as timed out when a connection with the server cannot be established. A value of zero never marks the agent as timed out.
+- `default_apps` (List of String) The list of built-in apps to display in the UI. Defaults to all apps.
```

Comment:
Thanks for this. This would be a great addition.
Should we also document a list of all available built-in `default-apps`?
We should document this somewhere in our docs. Probably on this page https://coder.com/docs/v2/latest/templates

Reply:
We can, the provider is self-documenting now that they're enumerated.